src/Object/Api/Mastodon/Status/FriendicaExtension.php

```diff
@@ -47,13 +47,16 @@ class FriendicaExtension extends BaseDataTransferObject
 
 	/** @var FriendicaDeliveryData|null */
 	protected $delivery_data;
+
 	/** @var int */
 	protected $dislikes_count;
 	/**
 	 * @var FriendicaVisibility|null
 	 */
 	protected $visibility;
 
+	/** @var bool */
+	protected $disliked = false;
```

Comment:
Please move this below `$dislikes_count` as per the constructor parameter order.